.circleci/config.yml

```diff
@@ -48,8 +66,12 @@ references:
 
   android_config: &android_config
     working_directory: *workspace
+    environment:
+#        GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xms1536m -Xmx4096m -XX:+HeapDumpOnOutOfMemoryError"'
```

Comment:
I did comment on commit #840562 message: a select android api 27 machine image to avoid an unnecessary platform update

Is an in-line comment desired as well?

Reply:
Indeed, there's no use in the process. I only found it convenient to leave it there, for easy toggling in future experiments